druid-shell/src/platform/x11/application.rs

```diff
@@ -24,6 +24,8 @@ use crate::application::AppHandler;
 use crate::kurbo::Point;
 use crate::{KeyCode, KeyModifiers, MouseButton, MouseEvent};
 
+use lazy_static::lazy_static;
```

Comment:
Not a big deal right now, but generally we use the following order with imports:
1. std modules
2. external modules
3. internal modules

Reply:
Re-ordered imports for all the files under `druid-shell`.

Though some files under `druid-derive` and `druid` don't follow this order!

Reply:
Thanks a lot! :+1:

Yes it's true this is not properly followed. My comment here and your response is part of the process in getting it fixed. :slightly_smiling_face:

Reply:
Sorry, I guess my comment may seem somewhat rough.
What I meant was that some files under `druid-drive` and `druid` do not follow import order and should be fixed.

I would love to fix those as well, maybe in this or other PR?

Reply:
There's nothing unpleasant, no worries. Yes you can send a new PR with those changes. An addition to the contribution guide is also welcome.

This PR here should just wait for merging now.